bin/compile

```diff
@@ -12,10 +12,7 @@ NGINX_VERSION=1.13.6
 PCRE_VERSION=8.41
 
 nginx_tarball_url=http://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz
-pcre_tarball_url=https://ftp.pcre.org/pub/pcre/pcre-${PCRE_VERSION}.tar.bz2
```

Comment:
Agreed these should be removed, but it looks like we are still attempting to download these below:

(line 31: `curl -L $ngx_postgres_url | tar xzv`)

as well as add their module to nginx:

(line 46) `add-module=ngx_postgres-1.0 \`